tidb-server/main.go

```diff
@@ -184,6 +189,30 @@ func exit() {
 	os.Exit(0)
 }
 
+func setCPUAffinity() {
+	if affinityCPU == nil || len(*affinityCPU) == 0 {
+		return
+	}
+	var cpu []int
+	for _, af := range strings.Split(*affinityCPU, ",") {
+		af = strings.TrimSpace(af)
+		if len(af) > 0 {
+			c, err := strconv.Atoi(af)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "wrong affinity cpu config: %s", *affinityCPU)
+				exit()
+			}
+			cpu = append(cpu, c)
+		}
+	}
+	err := linux.SetAffinity(cpu)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "set cpu affinity failure: %v", err)
+		exit()
+	}
+	runtime.GOMAXPROCS(len(cpu))
```

Comment:
Will it conflict with `max-procs`?

Reply:
good catch.. I have move this after `setGlobalVars`, so it will override `max-procs`..also remove reload max-procs function